static/js/containers/LoginButton.js

```diff
@@ -7,9 +7,23 @@ class LoginButton extends React.Component {
   render() {
     const { authentication } = this.props;
 
-    return <DropdownButton title={authentication.name}>
-      <MenuItem eventKey="logout">Logout</MenuItem>
-    </DropdownButton>;
+    return (
+      <DropdownButton
+        title={authentication.name}
+        id="logout-button">
+        {this.logOutButton()}
```

Comment:
totally possible, I tend to write a lot of small helper methods with React, but this isn't a super complicated component so it's probably fine to just have one big `render` method.